### ISSUE TITLE
implementation questions

### ISSUE THREAD

**Issue by world2vec** [131250208/JointExtraction#36]
Hi,
Thanks for your good idea and paper, have some questions about TPLinkerPlus
1. I see you use logsum to calculate the loss, is it  better than BCE?
2.  Is there any reference/paper for "conditional" layernorm which is used for "cln" shaking_type? 
Thanks.

**Comment by 131250208** [comment]
1. It brought a slight improvement. For more details, see [Extend "softmax+cross entropy" to multi-label classification problem](https://spaces.ac.cn/archives/7359)
2. For all I know, no. But you can refer to this blog [Conditional text generation based on Conditional Layer Normalization](https://spaces.ac.cn/archives/7124).

**Comment by world2vec** [comment]
Thanks for your quick response and guidance.
Looking forward to your paper to know the gains of these 2 component.
The first run of my implementation only have about 90.x on the NYT dataset without these 2 component(no hyper parameter tuning, just converge) which looks too low. I will double check the code and tuning a little. 
Thanks.
